appmap/django.py

```diff
@@ -254,6 +258,24 @@ def __call__(self, request):
             )
             self.recorder.add_event(return_event)
 
+        if self.recorder.enabled and Env.current.record_all_requests:
```

Comment:
How does this recording ensure that it captures only events from this request and not from other concurrent requests?

Reply:
It doesn't.  I plan to:
- add logic that deletes previous events assuming no two requests execute in same process
- start a Django and Flask webserver to verify this with a real server

Reply:
> assuming no two requests execute in same process

Isn't the webserver multi threaded?

I'm not sure what you mean by delete previous events. Previous to what? Those events might belong to a different request, so it's best to keep them organized into the proper recorder object as they come in.